### DISCUSSION TITLE
How to resize image in flex-grow zone ?

### DISCUSSION THREAD

**Discussion by Warrangel** [twbs/bootstrap#35003] [Q&A]
Hey, 

The goal of my project is to have a non-scrollable page.
I need to resize my image in a flex-grow area but the height of the image  is kept when the area shrinks so the scroll is visible and when the area is enlarged, it's ok. I can't fixe my area because  the area above is dynamic.

The problem exists for portrait images...

I share with you an explanatory diagram :
![unicorn_example](https://user-images.githubusercontent.com/4269056/133761342-3e0a586a-938f-47af-ad59-97539b72ee18.png)

Thank you !!!

**Comment by ghost** [comment]
https://getbootstrap.com/docs/5.1/content/images/#responsive-images

Did you checked this out?

**Reply by Warrangel** [reply]
i have already this class on my picture :/ .... so i think it's due to my parent who is in flex-grow and therefore not a 'fixed' size ... because if I add a height on my zone it works fine.
